scss/index.scss

```diff
@@ -235,6 +235,25 @@
   display: none;
 }
 
+.tos-section {
```

Comment:
These should be in a separate `.scss` file. You can add new .scss files to be compiled like so: https://github.com/morehumaninternet/roar-server/blob/37c7c656cd648849f8a29aaba689cc7757e84393/build.js#L38

Reply:
add separate scss file as suggested